contracts/Content/interfaces/IUniqueContent.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../libraries/LibAsset.sol";
+
+interface IUniqueContent {
+    /*********************** Events *********************/
+    event Mint(address operator, LibAsset.UniqueAssetCreateData data, uint256 uniqueId);
```

Comment:
For Burn and Mint events, move uniqueId as the first data to pass. so Mint(uniqueId, operator, data) and Burn(uniqueId, operator). Just so it's consistent with UniqueUriUpdated. Also, add "indexed" to the uniqueId